simulation/src/LeiosProtocol/Short.hs

```diff
@@ -801,6 +804,7 @@ endorseBlocksToVoteFor cfg@LeiosConfig{voteSendStage} slotConfig slot ibs ebs =
 -----------------------------------------------------------------
 ---- Expected generation rates in each slot.
 -----------------------------------------------------------------
+-- @bwbush: We may not need subslots because the sortition proposal limits producers to one IB or EB per slot. Even if stake where extremely concentrated, this would still be a good approximation.
```

Comment:
Is this recent? Last I heard, nodes were allowed to produce multiple IBs per slot (and frequently would, even if those extra IBs would be empty)

Reply:
We have discussed both approaches, so maybe we should discuss once more and resolve it. The CIP currently says . . .

> In Leios, an IB producer, EB producer, or voter is only allowed one production for each winning of the sortition lottery.

I believe this matches the Agda spec, too, but the paper seems to be silent on it. In practice a producer would only very rarely win more than once and only if they had a high concentration of stake.